### PR TITLE
Update OGI metadata

### PR DESCRIPTION
**Pull request by cthoyt**
1. Add Asiyah Yu Lin as contact @linikujp since on https://code.google.com/archive/p/ontology-for-genetic-interval/ she's listed as the first author for the publications
2. Switch status from orphaned to inactive (maybe also consider obsolete, as proposed in #1942, since this is on a defunct platform and doesn't enable standard OBO-required community interaction)

### CODE REVIEW

**Comment by matentzn** [comment]
So are you saying that the whole ontology was merged into https://obofoundry.org/ontology/ogsf.html?

If so, we should obsolete it and redirect users to OGSF instead. It is very important that an ID can always be clearly associated with one ontology. For example OGI:123 should either belong to OGI or OGSF, not both. Would you say that all off OGI is now managed by the OGSF, and that requests and issues should be raised on the OGSF repo? If so, I suggest we obsolete it and add "replaced by" OGSF. Users should always know where to find a term!

Are you ok to be listed as contact either way?

**Comment by linikujp** [comment]
Yes. I will be the contact.

**Comment by matentzn** [comment]
@linikujp can you comment on my question?

File: ontology/ogi.md [hunk 1]
```diff
@@ -3,11 +3,16 @@ layout: ontology_detail
...
+  label: Asiyah Yu Lin
+  email: linikujp@gmail.com
+  github: linikujp
+  orcid: 0000-0002-5379-5359
 products:
   - id: ogi.owl
 title: Ontology for genetic interval
 tracker: https://code.google.com/p/ontology-for-genetic-interval/issues/list
-activity_status: orphaned
+activity_status: inactive
```
Comment (matentzn) [inline ontology/ogi.md:15]: change to "obsolete" and add a replaced_by link to ogsf (as per email discussions with @linikujp)

Keep contact information the way you have it here!

**Review by matentzn — CHANGES_REQUESTED** [review]

File: ontology/ogi.md (hunk 1 above)
Comment (cthoyt) [reply ontology/ogi.md:15]: done, but obsolete is its own field unrelated to activity status so note that it is now marked as inactive

**Comment by matentzn** [comment]
Excellent. I will merge this on Monday. Thank you @cthoyt and @linikujp

**Comment by linikujp** [comment]
Yes. I can be the contact.
And Yes the OGI is merged into OGSF.

**Pull request merged**